### PR TITLE
Fix for ESBJAVA-3967

### PR DESCRIPTION
**Pull request by Shakila**

### CODE REVIEW

File: components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/java/org/wso2/carbon/inbound/ui/internal/InboundManagementClient.java [hunk 1]
```diff
@@ -406,4 +407,35 @@ public String getKAFKASpecialParameters() {
...
+        String topicListParams = "";
+        loadProperties();
+        if (prop != null) {
+            String strKey = "kafka.mandatory";
+            String strLength = prop.getProperty(strKey);
+            Integer iLength = null;
+            if (strLength != null) {
+                try {
+                    iLength = Integer.parseInt(strLength);
+                } catch (Exception e) {
```
Comment (RKathees) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/java/org/wso2/carbon/inbound/ui/internal/InboundManagementClient.java]: Why have we caught generic exceptions? can be used specific NumberFormatException

Comment (Shakila) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/java/org/wso2/carbon/inbound/ui/internal/InboundManagementClient.java]: https://github.com/wso2/carbon-mediation/pull/460

File: components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp [hunk 2]
```diff
@@ -76,11 +76,16 @@
...
                 }else if(strKey.startsWith("coordination")){
-		    sParams.add((new ParamDTO("coordination",request.getParameter("coordination"))));
-                }else if(strKey.startsWith("zookeeper.") || strKey.startsWith("group.id") || strKey.startsWith("auto.")|| strKey.startsWith("topic.filter.")|| strKey.equals("topics")||strKey.startsWith("filter.from.")||strKey.startsWith("consumer.type")|| strKey.startsWith("thread.count")|| strKey.startsWith("simple.")|| strKey.startsWith("content.type")){
-                  String strVal = request.getParameter(strKey);
+		            sParams.add((new ParamDTO("coordination",request.getParameter("coordination"))));
+                }else if(strKey.startsWith("zookeeper.") || strKey.startsWith("group.id") || strKey.startsWith("auto.")|| strKey.startsWith("topic.filter")|| strKey.equals("topics")||strKey.startsWith("filter.from")||strKey.startsWith("consumer.type")|| strKey.startsWith("thread.count")|| strKey.startsWith("simple.")|| strKey.startsWith("content.type")){
+                  String strVal = request.getParameter(strKey).trim();
+                  if(strKey.trim().equals("filter.from"))
+                  {
```
Comment (RKathees) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp]: Formating issue

File: components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/updateInbound.jsp [hunk 3]
```diff
@@ -74,11 +74,16 @@
...
                 }else if(strKey.startsWith("coordination")){
-		   sParams.add((new ParamDTO("coordination",request.getParameter("coordination")))); 
-		        } else if(strKey.startsWith("zookeeper.") || strKey.startsWith("group.id") || strKey.startsWith("auto.")|| strKey.startsWith("topic.filter.")|| strKey.equals("topics")||strKey.startsWith("filter.from.")||strKey.startsWith("consumer.type")|| strKey.startsWith("thread.count")|| strKey.startsWith("simple.")|| strKey.startsWith("content.type")){
-                   String strVal = request.getParameter(strKey);
+		           sParams.add((new ParamDTO("coordination",request.getParameter("coordination"))));
+		        } else if(strKey.startsWith("zookeeper.") || strKey.startsWith("group.id") || strKey.startsWith("auto.")|| strKey.startsWith("topic.filter")|| strKey.equals("topics")||strKey.startsWith("filter.from")||strKey.startsWith("consumer.type")|| strKey.startsWith("thread.count")|| strKey.startsWith("simple.")|| strKey.startsWith("content.type")){
+                   String strVal = request.getParameter(strKey).trim();
+                   if(strKey.trim().equals("filter.from"))
+                   {
```
Comment (RKathees) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/updateInbound.jsp]: Same goes here as well.

Comment (Shakila) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/updateInbound.jsp]: https://github.com/wso2/carbon-mediation/pull/460/

File: components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp (hunk 2 above)
Comment (Shakila) [inline components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/resources/web/inbound/saveInbound.jsp]: https://github.com/wso2/carbon-mediation/pull/460

**Pull request merged**